astropy/io/ascii/tests/t/cds/description/table.dat

```diff
@@ -0,0 +1,10 @@
+Cr110   2108 6696.79 Al1  4.02  -1.42  29.5   2.2  0.289
+Cr110   2108 6698.67 Al1  3.14  -1.65  58.0   2.0  0.325
+Cr110   2108 7361.57 Al1  4.02  -0.90  44.1   4.0  0.510
+Cr110   2108 7362.30 Al1  4.02  -0.75  62.7   3.9  0.577
+Cr110   2108 7835.31 Al1  4.02  -0.65  73.7   6.6  0.539
+Cr110   2108 7836.13 Al1  4.02  -0.49  87.6   4.1  0.390
+Cr110   2108 8772.86 Al1  4.02  -0.32  87.6   5.1  0.957
+Cr110   2108 8773.90 Al1  4.02  -0.16 118.6  14.6  0.736
+Cr110   2108 5853.67 Ba2  0.60  -1.00 121.9   5.5  1.435
+Cr110   2108 6141.71 Ba2  0.70  -0.08 191.0   8.7  1.117
```

Comment:
For the test it woul be totally sufficient to have 2 lines of data here, but this is fine with me, too.